src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -171,11 +171,7 @@
                                        {:content (fn []
                                                    [method-menu/view on-change-profile-pic])
                                         :shell?  true}]))
-              :image-picker-props  {:profile-picture     (or
-                                                          @profile-pic
-                                                          (rf/sub
-                                                           [:profile/onboarding-placeholder-avatar
-                                                            @profile-pic]))
+              :image-picker-props  {:profile-picture     @profile-pic
```

Comment:
Thanks @J-Son89  as I checked if you go to next screen (set password) and the click on back button you still can see the selected avatar photo. please let me know if i should check any other scenario 

https://github.com/status-im/status-mobile/assets/71308738/e52099ab-8c0b-4b76-84a4-1ba61f890d08

Reply:
hi, the `or` logic here is for rendering initial avatars when no image set 
https://github.com/status-im/status-mobile/blob/b70a6c300eef969a7665d22281ba98ec470d1efb/src/status_im2/subs/profile.cljs#L27

I believe this will break initial avatar

Reply:
The reason for the issue might be (just a guess) image server port changed when back to foreground. media server before login is started at https://github.com/status-im/status-go/blob/21ddaa4b9f57288ed99fed04db528f3790b7bd2f/node/get_status_node.go#L210

Reply:
@yqrashawn yes as i mentioned the issue is related to the port but i can't see any issue with this changes as `user-avatar` already handled initial avatar according to full-name.
```
(if (and full-name (not (or profile-picture-fn profile-picture)))
       [initials-avatar props]
...
```

Reply:
oh, i see